packages/blockchain-info-components/src/Images/Images.ts

```diff
@@ -124,6 +125,7 @@ const Images = {
   'allow-camera2': allowCamera2,
   'allow-camera3': allowCamera3,
   'apple-app-store-badge': appleAppStoreBadge,
+  'bg-illo': bgIllo,
```

Comment:
Illo?

Reply:
that is how design call it I just reuse name, but we can rename it to some bg-pattern or something.